src/actions/OtpActions.js

```diff
@@ -7,30 +7,20 @@ import type { Dispatch, GetState } from '../types/reduxTypes.js'
 export const enableOtp = () => async (dispatch: Dispatch, getState: GetState) => {
   const state = getState()
   const account = CORE_SELECTORS.getAccount(state)
-  try {
```

Comment:
We don't need try/catch here because the `dispatch` includes that by default for all actions